hdrop/Makefile

```diff
@@ -0,0 +1,20 @@
+DESTDIR ?= /
+PREFIX ?= $(DESTDIR)usr/local
+EXEC_PREFIX ?= $(PREFIX)
+DATAROOTDIR ?= $(PREFIX)/share
+BINDIR ?= $(EXEC_PREFIX)/bin
+MANDIR ?= $(DATAROOTDIR)/man
+MAN1DIR ?= $(MANDIR)/man1
+
+all: hdrop.1
+
+hdrop.1: hdrop.1.scd
+	scdoc < $< > $@
+
+install: hdrop.1 hdrop
+	@install -v -D -m 0644 hdrop.1 --target-directory "$(MAN1DIR)"
```

Comment:
Having this implies the existence of a manpage. Do you intend to write one?

Reply:
No, not for now. I tried to take it out, is it correct now?

Reply:
You can also remove any `hdrop.1` strings in there, including the entire `hdrop.1` build step.

Reply:
Ok i took everything out of `default.nix` and `Makefile`